### ISSUE TITLE
Gravatar options not longer working

### ISSUE THREAD

**Issue by LiaraAlis** [dereuromark/cakephp-tools#246]
Gravatar urls with multiple options are not longer working correctly.

Generated and printed url is: `http://www.gravatar.com/avatar/704dc961bc8552e94dfdae8d86e0bedf?size=25&amp;default=mm`

Correct is `http://www.gravatar.com/avatar/704dc961bc8552e94dfdae8d86e0bedf?size=25&default=mm`

**Comment by dereuromark** [comment]
Must be a recent core fix that encodes now.

**Comment by LiaraAlis** [comment]
Currently I'm using CakePHP 3.6.15. It is not the newest version and I can not say, why this is happening.
Before the update it worked perfectly.

**Comment by LiaraAlis** [comment]
Sure. Here is my code:
```php
<td>
    <?= $this->Gravatar->image($user->email, ['class' => 'img-circle', 'size' => 25, 'default' => 'mm']); ?>
    <?= $user->full_name ?>
    <?php if ($user->is_admin): ?>
        <small class="label label-danger"><?= __('Administrator') ?></small>
    <?php endif ?>
</td>
```

**Comment by dereuromark** [comment]
I see.
Do you have a chance to make a PR that fixes things?

**Comment by dereuromark** [comment]
So whats actually happening is not the normal escape which is fine, but a double escape:

    ?size=25&amp;amp;default=mm

That is the issue here, and you can see that opening the source code usually.

I have a fix here: https://github.com/dereuromark/cakephp-tools/pull/247
Please confirm :)

**Comment by dereuromark** [comment]
Do you know what Cake version or change caused this?
We need to bump composer now to that version - as otherwise the versions before are maybe faulty.

**Comment by LiaraAlis** [comment]
Hi,

I can not say this exactly. As I said before, I saw this problem after Upgrading CakePHP to 3.6.15.
Before I used CakePHP 3.4.3.

I'll try to check your fix as soon as I can.

**Comment by LiaraAlis** [comment]
Unfortunately, this doesn't fix the problem for me. Because I'm using CakePHP 3.6.x and the minimum for this package is actually 3.7. :(

**Comment by dereuromark** [comment]
At this time everyone should already be at 3.8 or at least 3.7. 3.6 is long dead.
And upgrade path is trivial in minors.

**Comment by LiaraAlis** [comment]
I think you're right. I will try to update to 3.7 in the next time.

**Comment by LiaraAlis** [comment]
I can confirm that the fix is working with CakePHP 3.7. ;)
